src/main/java/org/mockito/junit/VerificationCollectorImpl.java

```diff
@@ -52,6 +63,15 @@ public void collectAndReport() throws MockitoAssertionError {
         }
     }
 
+    public VerificationCollector assertAllLazily() {
```

Comment:
Laziness is not turned on by default. If we want to make this default with the rule, this method would not be needed.